files/scripts/saidump.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
please move this file to syncd/scripts

Reply:
OK. Fix it.

Reply:
@lguohan , please help to review it again. Thanks.

Reply:
Thanks for your comments! I will fix it.

Reply:
i should have be more clear. when i say syncd docker, i really means sairedis repo, so that saidump.sh will be part of the sairedis package.